package.json

```diff
@@ -16,41 +16,47 @@
     "LICENSES/*"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=12"
   },
   "dependencies": {
     "debug": "^4.3.2",
+    "pull-cat": "^1.1.11",
     "pull-pushable": "^2.2.0",
     "pull-stream": "^3.6.0",
     "ssb-db2": ">=3.0.0 <=6",
     "ssb-ebt": "^9.1.2",
-    "ssb-meta-feeds": "~0.35.2",
+    "ssb-meta-feeds": "^0.38.2",
     "ssb-network-errors": "^1.0.1",
-    "ssb-subset-ql": "^1.0.0"
+    "ssb-subset-ql": "^1.0.1",
+    "ssb-tribes2": "^0.3.0",
+    "ssb-box2": "^4.0.0"
   },
   "devDependencies": {
+    "c8": "^7.12.0",
     "cat-names": "^3.0.0",
     "dog-names": "^2.0.0",
+    "eslint": "^8.27.0",
     "husky": "^4.3.0",
     "mkdirp": "^1.0.4",
-    "c8": "^7.12.0",
     "prettier": "^2.4.0",
     "pretty-quick": "^3.1.0",
     "promisify-4loc": "^1.0.0",
     "pull-paramap": "^1.2.2",
     "rimraf": "^3.0.2",
-    "scuttle-testbot": "^1.10.1",
-    "secret-stack": "^6.4.1",
-    "ssb-bendy-butt": "^1.0.1",
-    "ssb-caps": "^1.1.0",
+    "scuttle-testbot": "^1.11.0",
+    "secret-stack": "6.4.1",
+    "ssb-bendy-butt": "1.0.2",
+    "ssb-caps": "1.1.0",
+    "ssb-conn": "6.0.4",
+    "ssb-classic": "1.1.0",
+    "ssb-box": "^1.0.1",
     "ssb-db": "^20.4.0",
-    "ssb-db2": "^6.2.4",
-    "ssb-conn": "^6.0.4",
-    "ssb-fixtures": "^3.0.4",
-    "ssb-friends": "^5.1.7",
+    "ssb-db2": "6.3.0",
```

Comment:
why are these hardcoded to specific versions now?

Reply:
Oh, `devDependencies` hardcoded makes sure that the test suite is always reproducible. I've had cases where something changed in a patch version and then other packages' tests broke. I don't remember if ssb-replication-scheduler suffers this but: I've had quite a lot of chicken and egg headaches with ssb-db2, e.g. ssb-fixtures depends on ssb-db2 and ssb-db2 depends on ssb-fixtures. ssb-about-self wants ssb-db2 as a peer dep but ssb-db2 has ssb-about-self as a dep.

`dependencies` could use ranges, because with lots of others involved, we want to avoid duplicate packages.

Reply:
If you want, these changes can be reverted. I guess there were made back in the days when we were desperate trying to eliminate possible culprits.

Reply:
Fair enough, I havn't really looked at all the changes, I was just wondering if db2 6.3.1 was needed or not :)

Reply:
Oh yeah, probably not needed for these tests to pass, but needed in the big picture. I updated it now to 6.3.1

Reply:
Great thanks :)